Added MQTTPerfHarness support, updated common build logic, fixed JMS …

aneeshayerrathota-ibm opened the pull request:
Added MQTTPerfHarness support, updated common build logic, fixed JMS JNDI by removing CORBA instance, and added prereq MQTT_Clients folder

aneeshayerrathota-ibm committed:
Added MQTTPerfHarness support, updated common build logic, fixed JMS JNDI by removing CORBA instance, and added prereq MQTT_Clients folder

stmassey reviewed (commented):
Hi Aneesha - Im going to leave most of the review to Paul as hes most familiar with the MQTT code logic. One query I would make is why you couldnt reuse the existing DefaultMessageFactory and DefaultDestinationFactory classes? Paul has pointed out that DefaultMessageFactory is different and creates a byte[] rather than a Message - so thats fine. 

For DefaultDestinationFactory - even your code references the one in the PerfHarness project, so I think thats safe to delete from the MQTT project.

It looks like my Jakarta changes were before migrating from Java 8, so you were correct in commenting out the CORBA line. Its not used in general testing.

In MQTTPerfHarness.java there is also a println: 
		System.out.println("PAUL");

Is the mqttProvider.java file required? Not sure its referenced anywhere. (git makes it difficult to search in PRs)

pharrishur commented:
As per Sam's comment above and after reviewing it myself, I would do the following befroe re-submitting the pull request:
1. Remove mqttProvider .java (empty, unreferenced interface) I compiled MQTT ok without this.
2. Remove DefaultDestinationFactory.properties form the MQTT project as I thnk it should use the one in PerfHarness anyway, and that's up to date with other additions.
3. The build files still reference 'MQTT31' rather than MQTT  -nothing should be left that says MQTT31
4. Remove the System.out.println("PAUL"); (oops)
5. 
Re-build and re-test that it still all works.

aneeshayerrathota-ibm committed:
Addressed Review Comments : removed mqttProvider and DefaultDestinationFactory.properties, clean build refs, removed debug print.

aneeshayerrathota-ibm commented:
Hi @pharrishur ,
I have addressed the review comments, and you should now see cleaner build refs and no debug comments. I have also removed unreferenced files.

Thank You

aneeshayerrathota-ibm committed:
Updated MQTTPerfHarness build scripts, removed old perfharness_mqtt.jar

aneeshayerrathota-ibm commented:
Hi @pharrishur ,
I have updated once more to correct the build jar name and to run it independently from MQTTPerfHarness as well.

pharrishur reviewed (approved):
Thanks Aneesha.
